resources/ory/charts/oathkeeper/templates/deployment-sidecar.yaml

```diff
@@ -30,6 +30,8 @@ spec:
         {{- toYaml . | nindent 8 }}
     {{- end }}
     spec:
+      seccompProfile:
```

Comment:
I think this is not valid here ([reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)) and should be moved to `spec.securityContext`.